src/raster/r.pops.spread/pops-core/CITATION.cff

```diff
@@ -16,6 +16,6 @@ authors:
 title: r.spread.sod - SOD spread model implemented as GRASS GIS module
 version: 29074a1
 date-released: 2018-06-21
-url: https://grass.osgeo.org/grass74/manuals/addons/r.spread.sod.html
+url: https://grass.osgeo.org/grass7/manuals/addons/r.spread.sod.html
```

Comment:
Since it is a citation suggestion, is this still valid?

Reply:
It seems that page got renamed? I cannot find "r.spread.sod.html"...

Reply:
Fixed in dbc78fa2c018604bd24a5e8c750aa6a8bc75b17f